TheMovieDB/TheMovieDB/Libraries/Facade.swift

```diff
@@ -20,5 +21,8 @@ final class Facade {
     func retrieveData(result: @escaping (Result<intermediaryJson, Error>) -> Void) {
         networkManager.retrieveMovies(completion: result)
     }
+    func loadImgFromCache(_ urlString: String, result: @escaping (Result<UIImage, Error>) -> Void) {
```

Comment:
name this method in a different way, remember that the facade is should expose just the behaviour not the implementation. This is telling from where it is taking the image, you could name it just like "loadMovieImage" that way you are hiding the implementation details.